Packaging missing critical files

Hello friend! I wanted to try your package but it is not built correctly

`npm install syncguard@latest @google-cloud/firestore`

```
$ ls node_modules/syncguard/
LICENSE  README.md  index.d.ts  index.js  package.json
```

Please note the lack of common/

```
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/app/node_modules/syncguard/common/backend.js' imported from /app/node_modules/syncguard/index.js\n  
```

--- a/package.json
+++ b/package.json
@@ -43,8 +43,8 @@
   "module": "./index.js",
   "types": "./index.d.ts",
   "files": [
-    "*.js",
-    "*.d.ts",
+    "**/*.js",
+    "**/*.d.ts",
     "LICENSE",
     "README.md"
   ],

